tests/perf/fair_queue_perf.cc

```diff
@@ -75,10 +75,10 @@ struct perf_fair_queue {
 
     seastar::sharded<local_fq_and_class> local_fq;
 
-    seastar::fair_group shared_fg;
+    seastar::shared_throttle shared_fg;
 
-    static fair_group::config fg_config() {
-        fair_group::config cfg;
+    static shared_throttle::config fg_config() {
+        shared_throttle::config cfg;
         return cfg;
     }
```

Comment:
Minor: the same question about changing `fgs` -> `throttle` in the next patch and usage of `fg` name in this file.

Reply:
Okay, please ignore the comment -- I see that at the end of patches we remove it completely.